addons/html_editor/__manifest__.py

```diff
@@ -17,11 +17,13 @@
     'auto_install': True,
```

Comment:
Did not you say this PR targets 18.0 because it restores a behavior that was there in the past? Could you add the relevant information about that in the commit message(s) please? :pray:

Reply:
yes, I add it to the commit message :pray: